src/deploy-cromwell-on-azure/scripts/cromwell-application.conf

```diff
@@ -36,9 +51,17 @@ backend {
       actor-factory = "cromwell.backend.impl.tes.TesBackendLifecycleActorFactory"
       config {
         filesystems {
-          http { }
+              http {
+                enabled: true
+              }
+              local {
+                enabled: true
```

Comment:
should this still be enabled in addition to `blob`

Reply:
Yeah, local still needs to be enabled as long as we have the blobfuse mounts.